packages/ui/components/core/VerifiedBadge.tsx

```diff
@@ -0,0 +1,65 @@
+import { Icon } from '@iconify/react'
+import { Avatar, Tooltip, Text, createStyles, Badge } from '@mantine/core'
+import { DateTime } from 'luxon'
+import { useTranslation } from 'next-i18next'
+// tooltip max characters for width limit
+const MAX_CHARACTERS = 80
+
+const useStyles = createStyles((theme) => ({
+	avatar: {},
+	badge: {
+		paddingLeft: 0,
+		paddingRight: 0,
+		borderStyle: 'hidden',
+	},
+	icon: {
+		color: 'white',
+		position: 'absolute',
+		fontWeight: theme.other.fontWeight.semibold,
+		height: '17.5px',
+		width: '17.5px',
+	},
+	text: {
+		width: 'auto',
+		fontWeight: theme.other.fontWeight.semibold,
+		color: 'black',
+		marginLeft: '9.25px',
+	},
+	tooltip: {},
+}))
+
+export const VerifiedBadge = ({ lastVerifiedDate }: Props) => {
+	const { classes } = useStyles()
+	const { t, i18n } = useTranslation()
+
+	const dateString = DateTime.fromJSDate(lastVerifiedDate)
+		.setLocale(i18n.resolvedLanguage)
+		.toLocaleString(DateTime.DATE_MED_WITH_WEEKDAY)
+	const tooltipText = t('verified-information-detail', { dateString })
```

Comment:
passing that formatted date string in as a param - gets inserted in the `{{dateString}}` placeholder in the JSON file.